app/src/main/java/org/schabi/newpipe/player/notification/NotificationConstants.java

```diff
@@ -181,7 +180,7 @@ public static List<Integer> getCompactSlotsFromPreferences(
 
             if (compactSlot == Integer.MAX_VALUE) {
                 // settings not yet populated, return default values
-                return new ArrayList<>(Arrays.asList(SLOT_COMPACT_DEFAULTS));
+                return SLOT_COMPACT_DEFAULTS;
```

Comment:
The array needs to be mutable, as it will be mutated in `NotificationActionsPreference`.
```suggestion
                return new ArrayList<>(SLOT_COMPACT_DEFAULTS);
```

Or remove all `new ArrayList<>` from this function and move them to `NotificationActionsPreference`